src/markdown-pages/docs/first-steps/installing-offline.md

```diff
@@ -35,6 +35,31 @@ Depending on how you use Galasa, there are several software prerequisites, some
 | 3270 emulator | Optional. Although you do not need a 3270 emulator to run a Galasa test (even if it tests a 3270 application) you can use one to explore Galasa Simbank, a simulated version of an application that helps you get acquainted with Galasa before connecting to a real mainframe to run your own tests. There are many such emulators available but IBM's Personal Communications (PCOMM) is frequently used, as is IBM's Host on Demand software, which includes support for Windows, Linux and MacOS.| 
 
 
+## Installing Java 
+
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
```

Comment:
duplicate information with above file ? I've seen this before in this review...

Reply:
took this out in the end - see earlier comment